lib/helpers.ts

```diff
@@ -52,16 +50,22 @@ export function isPathIgnored(filePath: ?string, ignoredGlob: string, ignoredVCS
   return minimatch(normalizedFilePath, ignoredGlob)
 }
 
-export function subscriptiveObserve(object: Object, eventName: string, callback: Function): Disposable {
+export function subscriptiveObserve(
+  object: {
+    [key: string]: any
```

Comment:
Fyi the TS recommended way is `Record<string, any>` for this

Reply:
Yes, thanks. That's what flow-to-ts generates. We can fine-tune it later manually for sure.

Reply:
Fixed in https://github.com/steelbrain/linter/pull/1718/commits/9b053512126d477d2560502857a133f6ce3b1314

Reply:
Opened an issue for it: 

https://github.com/Khan/flow-to-ts/issues/198

Reply:
Thanks a lot!